bin/flutter

```diff
@@ -41,7 +41,25 @@ DART="$DART_SDK_PATH/bin/dart"
 
     echo Building flutter tool...
     FLUTTER_DIR="$FLUTTER_ROOT/packages/flutter"
-    (cd "$FLUTTER_TOOLS_DIR"; "../../bin/cache/dart-sdk/bin/pub" get --verbosity=warning)
+
+    # Capture pub output for filtering expected analyzer dependency warning.
+    PUB_OUT=`({(cd "$FLUTTER_TOOLS_DIR"; "../../bin/cache/dart-sdk/bin/pub" get --verbosity=warning)} 2>&1)`
+    ISSUE_COUNT=`(grep -o "Warning:\|Error:" <<< $PUB_OUT | wc -l)`
+    if [ $ISSUE_COUNT -gt 0 ]; then
```

Comment:
Also, I could be wrong but since we're running pub with the ` --verbosity=warning` flag, I think the only messages we should expect will be errors or warnings so I don't think there's anything to miss?